languages/elixir/exercises/concept/boutique-inventory/.docs/hints.md

```diff
@@ -20,8 +20,8 @@
 
 ## 4. Calculate the item's total quantity
 
-- Maps are enumerables too.
-- Iterating over a map will convert the map to a list of `{key, value}` tuples.
+- Maps implement the enumerable protocol.
+- `Enum` functions convert maps to a list of `{key, value}` tuples.
```

Comment:
The previous step had exactly the same hints but phrased totally differently for no reason 🤷 I'm making them the same.